bundle/src/main/java/com/adobe/acs/commons/wcm/impl/SiteMapServlet.java

```diff
@@ -298,8 +305,25 @@ private void write(Page page, XMLStreamWriter stream, ResourceResolver resolver)
         stream.writeEndElement();
     }
 
-    private boolean isHidden(final Page page) {
-        return page.getProperties().get(this.excludeFromSiteMapProperty, false);
+    private boolean isHiddenOrProtected(Page page) {
+        boolean flag= false;
+        if(this.excludeFromSiteMapProperty != null){
+            String [] multipleProperties = this.excludeFromSiteMapProperty.split(",");
```

Comment:
This still needs updating to @justinedelson 's suggestions of making the osgi property a multi-value, which means you dont need to split on commas.

Reply:
Yes, its done now